code-studio/src/js/pd/workshop_dashboard/attendance/workshop_attendance.jsx

```diff
@@ -0,0 +1,194 @@
+/* global React */
+
+var SessionTime = require('../components/session_time.jsx');
+var SessionAttendance = require('./session_attendance.jsx');
+var Row = require('react-bootstrap').Row;
+var ButtonToolbar = require('react-bootstrap').ButtonToolbar;
+var Button = require('react-bootstrap').Button;
+var Tabs = require('react-bootstrap').Tabs;
+var Tab = require('react-bootstrap').Tab;
+var Col = require('react-bootstrap').Col;
+
+var WorkshopAttendance = React.createClass({
+  contextTypes: {
+    router: React.PropTypes.object.isRequired
+  },
+
+  propTypes: {
+    params: React.PropTypes.shape({
+      workshopId: React.PropTypes.string.isRequired,
+      sessionIndex: React.PropTypes.string
+    }).isRequired
+  },
+
+  getInitialState: function () {
+    return {
+      loading: true
```

Comment:
I like to initialize all state to make it easier to track down what sorts of things we set, even if we just initialize to undefined (i.e. I'd add `workshopState: undefined, sessionAttendances: undefined`, etc)